pom.xml

```diff
@@ -4,19 +4,16 @@
   <parent>
     <groupId>org.nuxeo</groupId>
     <artifactId>nuxeo-parent</artifactId>
-    <version>2021.24</version>
+    <version>2021.27</version>
```

Comment:
Why not aligning on the latest version?

Reply:
I just applied the non-written policy about using the least version needed by the addon.

For the explorer case we can use last one if prefered.

Reply:
It has been update to `2021.41`.

Reply:
In the end I rollbacked to `2021.24` due to a lot of tests failures because bundles or components have changed between the two versions, and the required test code change should not be part of this utility task + I previously update the nuxeo-parent version to benefit from maven docker plugin update but as I removed these modules here I don't need it anymore.